7.Product_Designer-FlexBox/css/phone-screen.css

```diff
@@ -0,0 +1,55 @@
+/* ================================TITLE CONTENT=================================== */
+.designer{
+    flex-direction:column-reverse;
+    row-gap:6px;
+}
+/* Designer Name */
+.designer-name{
+    row-gap: 12px;
+}
+.designer-name h1{
+    font-size: 50px;
+}
+.designer-name h3{
+    font-size: 30px;
+}
+/* Designer Mail */
+.designer-mail{
+    justify-content: center;
+    font-size: 16px;
+    font-weight: 600;
+}
+
+/* =================================GALLERY CONTENT================================ */
+.gallery{
+    padding-left:20px;
+    padding-right:20px;
```

Comment:
Provide the shorthand notation of padding to reduce the number of lines
Eg.., padding: 0 20px;